files/en-us/web/css/css_anchor_positioning/using/index.md

```diff
@@ -156,20 +154,34 @@ Conventional absolutely and fixed positioned elements are explicitly positioned
-anchor(anchor-element anchor-side, length-percentage)
+anchor(<anchor-element> <anchor-side>, <fallback>)
 ```
 
 - `anchor-element`
-  - : The`<dashed-ident>` name set as the value of the [`anchor-name`](/en-US/docs/Web/CSS/anchor-name) property of the anchor element you want to position the element relative to. If omitted, the element's **default anchor**, which is the anchor referenced in the [`position-anchor`](/en-US/docs/Web/CSS/position-anchor) property, is used.
+
+  - : The`<dashed-ident>` value of the [`anchor-name`](/en-US/docs/Web/CSS/anchor-name) property of the anchor element you want to position the element relative to. If omitted, the element's **default anchor**, which is the anchor referenced in its [`position-anchor`](/en-US/docs/Web/CSS/position-anchor) property, is used.
+
 - [`<anchor-side>`](/en-US/docs/Web/CSS/anchor#anchor-side)
-  - : Specifies the physical (`top`, `left`, etc.) or logical (`start`, `end`, etc.) side of the anchor element that the positioned element will be positioned relative to. If this is not on the same axis as the inset property on which it is set, the fallback value is used.
-- {{cssxref("length-percentage")}}
-  - : Specifies the distance to use as a fallback value if the element is not absolutely or fixed positioned, if the axis of the anchor side is not valid for the property on which the `anchor()` function is set, or the anchor element doesn't exist.
 
-The most common `anchor()` parameters you'll use will refer to a side of the default anchor. You will also often add {{cssxref("margin")}} to create spacing between the edge of the anchor and positioned element.
+  - : Specifies the position relative to the corresponding side of the anchor. Valid values include the `center` of the anchor, other physical (`top`, `left`, etc.) or logical (`start`, `self-end`, etc.) sides of the anchor element, `inside` or `outside` the anchor's side, or a `<percentage>` between the start (`0%`) and end (`100%`) of the anchor's inset property's side. If a physical or logical value is used which is not on the same axis as the inset property on which it is set, the fallback value is used.
+
+- `<fallback>`
+
+  - : Specified as a {{cssxref("length-percentage")}} defining the distance to use as a fallback value if the element is not absolutely or fixed positioned, if the axis of the anchor side is not valid for the property on which the `anchor()` function is set, or if the anchor element doesn't exist.
+
+The return value of the `anchor()` function is a length value calculated based on the position of the anchor. If you set a length or percentage directly on an anchor-positioned element's inset property, the value is relative to where the element is in the source code, where it would be placed with that value if it were not bound to the anchor element. This is the behavior seen if the anchor-side is invalid for the inset property on which it is set and the fallback is used. These two declarations are equivalent:
```

Comment:
we never really explain what happens if anchor() isn't used. this was my attempt.
basically, anchor() does a lot of math, so we don't have to.